editor/src/core/performance/performance-regression-tests.spec.tsx

```diff
@@ -117,7 +117,7 @@ describe('React Render Count Tests - ', () => {
     )
 
     const renderCountAfter = renderResult.getNumberOfRenders()
-    expect(renderCountAfter - renderCountBefore).toBeGreaterThanOrEqual(455) // if this breaks, GREAT NEWS but update the test please :)
-    expect(renderCountAfter - renderCountBefore).toBeLessThan(465)
+    expect(renderCountAfter - renderCountBefore).toBeGreaterThanOrEqual(460) // if this breaks, GREAT NEWS but update the test please :)
```

Comment:
Interesting that this one has gone up...

Reply:
now that you mention – I didn't implement setState's default behavior that if `currentState === nextState` then `setState(nextState)` will be a NO_OP. I think I should amend this!